File: DirtBikePark/Models/Park.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DirtBikePark.Models
{
    public class Park
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Location { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public string ImageUrl { get; set; } = string.Empty;
        [Precision(18, 2)]
        public decimal PricePerAdult { get; set; }
        [Precision(18, 2)]
        public decimal PricePerChild { get; set; }
        public int GuestLimit { get; set; }
    }
}

File: DirtBikeFrontEnd/src/models/park.ts
import Review from "./review";

export default interface ParkInput
{
    name: string;
    location: string;
    description: string;
    imageUrl?: string;
    pricePerAdult: number;
    pricePerChild: number;
    guestLimit: number;
}

export default interface ParkResponse
{
    id: number;
    name: string;
    location: string;
    description: string;
    imageUrl?: string;
    pricePerAdult: number;
    pricePerChild: number;
    guestLimit: number;
    reviews: Review[];  // Mocked by service, do not exist in API
}